docker-compose.yml

```diff
@@ -0,0 +1,25 @@
+version: "3.8"
+
+services:
+  package:
+    image: node:16
```

Comment:
Node 18 is not the current active one. 

> Major Node.js versions enter Current release status for six months, **which gives library authors time to add support for them**. After six months, odd-numbered releases (9, 11, etc.) become unsupported, and even-numbered releases (10, 12, etc.) move to Active LTS status and are ready for general use. LTS release status is "long-term support", which typically guarantees that critical bugs will be fixed for a total of 30 months. Production applications should only use Active LTS or Maintenance LTS releases.

I avoided choosing 18 since we want to ensure that we are also working with the current `active` version (16). If someone adds code that only works with node 18 and then runs the tests with docker, it will work but someone else that uses the library in a node 16 environment will be met with errors.

Thus I prefer waiting for the active version to be 18, as that marks officially the moment where your application should be compatible with node 18.